uast/types.go

```diff
@@ -239,6 +239,7 @@ func fieldName(f reflect.StructField) (string, bool, error) {
 var (
 	reflString  = reflect.TypeOf("")
 	reflAny     = reflect.TypeOf((*Any)(nil)).Elem()
+	reflAnyArr  = reflect.TypeOf([]Any{})
```

Comment:
Nit: This is a slice, not an array. The difference is significant since an array type and a slice type do not reflect to the same mirror. I suggest: `reflAnySlice`.

Reply:
Right, arrays are used so rarely that I usually neglect the difference when it comes to naming. But yes, it matters here because of the reflection.